observability/otel/otel-forwarding-logs.adoc

```diff
@@ -0,0 +1,202 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="otel-forwarding-logs"]
+= Forwarding Logs to a LokiStack instance
+include::_attributes/common-attributes.adoc[]
+:context: otel-forwarding-logs
+
+You can deploy and configure the OpenTelemetry Collector to forward logs to a LokiStack instance. You can deploy the OpenTelemetry Collector in the deployment mode by using the specified processors, receivers, and exporters.
+
+This use of the Loki Exporter is a temporary Technology Preview feature, which is planned to be deprecated with the publication of an improved solution in which the Loki Exporter is replaced with the OTLP HTTP Exporter.
```

Comment:
@pavolloffay, if it's okay, I'm avoiding mentiong the fact "when the OpenShift LokiStack adds supports for OTLP HTTP" so that we don't commit to that date automatically because it will take some time after the OTLP HTTP support in LokiStack becomes available and our QE and docs become available.
https://github.com/openshift/openshift-docs/pull/80200#discussion_r1741832033